lib/iris/_data_manager.py

```diff
@@ -387,7 +393,7 @@ def shape(self):
         """
         return self.core_data.shape
 
-    def copy(self, data=None, realised_dtype=None):
+    def copy(self, data=None, realised_dtype='none'):
```

Comment:
This quite suble ...

We need a way to differential between the following three situations:

1. copy and use the current value of `realised_dtype`
1. copy and replace or "clear" the current value of `realised_dtype` i.e. set it to `None`
1. copy and replace the current value of `realised_dtype` i.e. set it to the value provided.

The use of `'none'` allows us to do this. 

Calling with no `realised_dtype` set i.e. default of `realised_dtype='none'`, is case 1.

Calling with `realised_dtype=None` is case 2.

Calling with `realised_dtype=X` is case 3.

Leaving the call signature with `..., realised_dtype=None)` does not support the above three required use cases.